components/public-api-server/pkg/apiv1/user.go

```diff
@@ -38,6 +40,9 @@ func (s *UserService) GetAuthenticatedUser(ctx context.Context, req *connect.Req
 	if err != nil {
 		return nil, proxy.ConvertError(err)
 	}
+	log.AddFields(ctx, logrus.Fields{
+		"user.id": user.ID,
```

Comment:
w.r.t to log-shapes: so far we follow [this format](https://github.com/gitpod-io/gitpod/blob/478a75e744a642d9b764de37cfae655bc8b29dd5/components/common-go/log/log.go#L35-L45), which is the same across logs and traces, in TS and Go (in most places :upside_down_face: ).
Not particularly attached to that specific format, but would be great to stay consistent within the app. Plus the existing format is imprinted into quite some :brain:s already :wink:

Reply:
Using the `.` format means it auto-unpicks them as nested properties of these structures, so they get treated more as JSON, rather than as plain strings

Reply:
Basically, in a log viewer you see it as "user": { "id": val } which means you group the properties of a user together

Reply:
I get that there are benefits to either side. I just want us to stay consistent, so I don't have to recall multiple formats.

Do you see any possibility we can achieve both? Like, fill multiple fields? :shrug:

Reply:
With this change, the logging in `public-api` is "standardised" to the `.` format. I don't really see much value in duplicating the fields currently. We can update server to use a similar format as well